### ISSUE TITLE
508-defect-3 [COGNITION]: "Sign up now" link should be styled as a link using the Action Link component

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22282]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

## Details

The "Sign up now" link is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click sign up now _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

## Acceptance Criteria

- [ ] Links are styled as links

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution (if known)

Use the new design system Action Link component
https://design.va.gov/components/action-links

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/112845307-b870be00-9072-11eb-9bf2-44425a0dc9df.png)
